wp-media/wp-rocket
# Closes #6978: exclude svg use from lrc

# Description

Fixes #6978
`SVG Use` will be excluded from LRC.

## Type of change


## Detailed scenario

- Be on latest version of WP Rocket
- Clear Cache + priority elements
- Load the page https://new.rocketlabsqa.ovh/lrc_live_3-17/
- You should see an infinite loading screen
- Clear the cache
- Reload the page
- Loading screen disappear and while scrolling you should see a corrupted logo
- 
## Technical description

### Documentation

We avoid adding hashes to containers that contains a `svg use` case. 

### New dependencies

Updated dependency: `rocket-scripts`.

### Risks
None

# Mandatory Checklist

## Code validation


## Code style
